custom/icds/location_rationalization/forms.py

```diff
@@ -0,0 +1,46 @@
+from django import forms
+from django.utils.translation import ugettext as _
+from django.utils.translation import ugettext_lazy
+
+from crispy_forms import layout as crispy
+from crispy_forms.bootstrap import StrictButton
+
+from corehq.apps.hqwebapp.crispy import HQFormHelper
+from corehq.util.workbook_json.excel import get_workbook
+
+
+class LocationRationalizationRequestForm(forms.Form):
+    file = forms.FileField(label="", required=True,
+                           help_text=ugettext_lazy("Upload xlsx file"))
```

Comment:
This UI needs more help text. Did you consider using the standard bulk download + re-upload page instead of making two pages? You'd need to customize it a bit to include the location & location type inputs, but it'd be better to use a standard workflow than to make a new one.

I have a few more UI comments that will be irrelevant if you use the standard bulk upload (consistency between the upload & download page, js spinner on submit, etc.).

Reply:
Resolved by https://github.com/dimagi/commcare-hq/pull/26892